templates/app/routes/route.hbs

```diff
@@ -3,50 +3,50 @@ const { PermissionMiddlewareCreator } = require('forest-express-{{#if isMongoDB
 const { {{ modelName }} } = require('../models');
 
 const router = express.Router();
-const permissionMiddlewareCreator = new PermissionMiddlewareCreator('{{ modelName }}');
+const permissionMiddlewareCreator = new PermissionMiddlewareCreator('{{ table }}');
 
-// This file contains the logic of every route in Forest Admin for the collection {{ modelName }}:
+// This file contains the logic of every route in Forest Admin for the collection {{ table }}:
 // - Native routes are already generated but can be extended/overriden - Learn how to extend a route here: https://docs.forestadmin.com/documentation/v/v5/reference-guide/routes/extend-a-route
 // - Smart action routes will need to be added as you create new Smart Actions - Learn how to create a Smart Action here: https://docs.forestadmin.com/documentation/v/v5/reference-guide/actions/create-and-manage-smart-actions
 
 // Create a {{ modelNameReadableSingular }}
-router.post('/{{ modelName }}', permissionMiddlewareCreator.create(), (request, response, next) => {
+router.post('/{{ modelNameDasherized }}', permissionMiddlewareCreator.create(), (request, response, next) => {
```

Comment:
Is it part of the issue?
Why should we have to change the route names?

Reply:
No, my bad. I revert this.